docs/root/modules/ontology/config.md

```diff
@@ -2,5 +2,13 @@
 
 Ontology does not require any specific configuration parameters. However, to enable a more controlled creation of ontology nodes, you can specify sources of truth for certain ontology node types.
 
+### Semantic Labels (Automatic)
+
+For modules that use semantic labels (like `UserAccount`), ontology properties are added automatically during node creation. No additional configuration is required - the ontology mappings define which properties are added to nodes.
```

Comment:
Could be good to link to the schema.md document that lists out all of the authoritative semantic labels.